src/I18n/I18n.php

```diff
@@ -71,7 +72,12 @@ public static function translators(): TranslatorRegistry
         );
 
         if (class_exists(Cache::class)) {
-            static::$_collection->setCacher(Cache::pool('_cake_core_'));
+            try {
+                $pool = Cache::pool('_cake_translations_');
+            } catch (InvalidArgumentException) {
+                $pool = Cache::pool('_cake_core_');
```

Comment:
There should be a deprecation warning after this statement informing the dev to update the config name.